### ISSUE TITLE
Random Softlock Issue

### ISSUE THREAD

**Issue by mnsg23** [jam1garner/smash-minecraft-skins#36]
**Issue**

I recently came across a random softlock, which occurred the instant I selected Steve on the character select screen. The screen itself froze, and yet the music was still playing. But after a while, the game ended up crashing, and I got a series of crash reports as a result.

**Crash Report Files**

[01608001819_01006a800016e000.log](https://github.com/jam1garner/smash-minecraft-skins/files/5693083/01608001819_01006a800016e000.log)
[01608001820_01006a800016e000.log](https://github.com/jam1garner/smash-minecraft-skins/files/5693084/01608001820_01006a800016e000.log)
[01608001821_01006a800016e000.log](https://github.com/jam1garner/smash-minecraft-skins/files/5693085/01608001821_01006a800016e000.log)
[01608001822_01006a800016e000.log](https://github.com/jam1garner/smash-minecraft-skins/files/5693086/01608001822_01006a800016e000.log)
[01608001825_01006a800016e000.log](https://github.com/jam1garner/smash-minecraft-skins/files/5693087/01608001825_01006a800016e000.log)
[01608001829_01006a800016e000.log](https://github.com/jam1garner/smash-minecraft-skins/files/5693088/01608001829_01006a800016e000.log)
[01608001835_01006a800016e000.log](https://github.com/jam1garner/smash-minecraft-skins/files/5693089/01608001835_01006a800016e000.log)

**System and Tools Used**

1. System Version: 11.0.1
2. Atmosphere Version: [0.16.1 (pre-release build)](https://github.com/Atmosphere-NX/Atmosphere/releases/tag/0.16.1)
3. ARCropolis Version: [v0.9.4](https://github.com/Raytwo/ARCropolis/releases/tag/v0.9.4)
4. smash-minecraft-skins Version: [1.1.3](https://github.com/jam1garner/smash-minecraft-skins/releases/tag/1.1.3)

**Comment by Lukayaba** [comment]
I'm also having this issue on the updated version

**Comment by Klingkoi** [comment]
Yup, got this issue multiple times and even managed to get this error.
![IMG_5609](https://user-images.githubusercontent.com/83049173/115808981-dfd85380-a3b0-11eb-9be6-7d6c9f4e19c5.JPG)

**Comment by Graxornam** [comment]
I am experiencing a softlock too, repeatedly, any time the skin selector is shown (which has the gall to present itself before the smash intro cutscene).
1. System Version: 12.0.1
2. Software Version: 10.1.0
3. Atmosphere Version: M.19.1
4. ARCropolis Version: v1.1.3
5. smash-minecraft-skins Version: 1.2.1
6. Fizaeu 2.1.5
7. Tesla 1.1.3
8. nx-ovlloader 1.0.6

**Comment by Lukayaba** [comment]
I heard someone say in a discord it has to do with overloads since we have tesla too? maybe deactivating it might help

Maybe overclocking too but I don't know shit about this
